R/scream.R

```diff
@@ -238,13 +238,13 @@ check_novel_levels <- function(x, ptype, column) {
 }
 
 warn_novel_levels <- function(levels, column) {
-  message <- glue(
-    "Novel levels found in column '{column}': {glue_quote_collapse(levels)}. ",
-    "The levels have been removed, and values have been coerced to 'NA'."
-  )
-
-  warn(
-    message,
+  n_levels <- length(levels)
+  cli::cli_warn(
+    c(
+      "{cli::qty(n_levels)}Novel level{?s} found in column {.field {column}}: {.val {levels}}.",
+      "i" = "The {cli::qty(n_levels)}level{?s} {?has/have} been removed,
```

Comment:
Indentation is difficult with these multiline cli statements. I have been lining up the text and element beginnings, such as 

```r
cli::cli_warn(
  c(
          "{cli::qty(n_levels)}Novel level{?s} found in column {.field {column}}: 
           {.val {levels}}.",
    "i" = "The {cli::qty(n_levels)}level{?s} {?has/have} been removed, and 
           values have been coerced to {.val NA}."
  ),
```

No mandate for any format here as long as it is consistent (within-package, at least)

Reply:
oh yeah ⚡ done in [3b61e62](https://github.com/tidymodels/hardhat/pull/272/commits/3b61e62e513910d9f57d2a428e0239cd6b3b42ea)